include/psp2/sblacmgr.h

```diff
@@ -0,0 +1,22 @@
+/**
+ * \usergroup{SceSblACMgr}
+ * \usage{psp2/sblacmgr.h,SceSblACMgr_stub}
+ */
+
+
+#ifndef _PSP2_SBLACMGR_H_
+#define _PSP2_SBLACMGR_H_
+
+#include <psp2/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int _sceSblACMgrIsGameProgram(void);

```

Comment:
can you remove underscore?

Reply:
no because sceSblACMgrIsGameProgram (without underscore) is a wrapper to this function.